### PR TITLE
Allow amp-ad data-loading-strategy attribute to take a float value.

### LINKED ISSUE #8199: amp-ad: Make viewport distance for prefer-viewability-over-views configurable
Currently, the `data-loading-strategy` option for `prefer-viewability-over-views` by default loads the ads only within 1.25 viewports of the current viewport. If publishers want to, we should allow this number to be configurable so publishers can make ads load more eagerly or lazily.
CC @cramforce

### PR DESCRIPTION
**Pull request by lannka**
Closes #8199

### CODE REVIEW

**Comment by lannka** [comment]
@jasti please take a look at the documentation change.

File: extensions/amp-ad/amp-ad.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ An optional attribute. If provided, will require confirming the [amp-user-notifi
 
 **data-loading-strategy**
 
-Instructs AMP to load ads in a way that prefers a high degree of viewability, while sometimes loading too late to generate a view. Supported value: `prefer-viewability-over-views`.
+An optional attribute that takes a float value in range of `[0, 3]`, which instructs the ad to start load when it's within the given number of viewports away from the current viewport. The default value is 3. Use a smaller value to gain higher degree of viewability, with the risk of generating less views.
```
Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: Having both `[0, 3]` and `prefer-viewability-over-views` in spec is confusing. For backward compatibility, we still support this string, but we don't recommend new users to use it.

If you want to talk about why people want `1.25` over the other random value, please suggest the wording.

Comment (jasti) [inline extensions/amp-ad/amp-ad.md]: "An optional attribute that takes a float value in range of `[0, 3]`, which instructs the ad to start loading when it's within the given number of viewports away from the current viewport. Use a smaller value to gain higher degree of viewability, with the risk of generating less views. If the attribute is not used, the default value is 3. If the attribute is used but the value is left blank, then a float value is assigned by the system which optimizes for viewability without drastically impacting the views."

Comment (jasti) [inline extensions/amp-ad/amp-ad.md]: @lannka Small change - "An optional attribute that takes a float value in range of [0, 3], which instructs the ad to start loading when it's within the given number of viewports away from the current viewport. Use a smaller value to gain higher degree of viewability, with the risk of generating fewer views. If the attribute is not used, the default value is 3. If the attribute is used but the value is left blank, then a float value is assigned by the system which optimizes for viewability without drastically impacting the views."

Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: updated code to reflect the logic

**Comment by lannka** [comment]
PTAL

**Pull request merged**